selfdrive/car/toyota/interface.py

```diff
@@ -265,21 +265,23 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
     elif candidate in [CAR.COROLLA_TSS2, CAR.COROLLAH_TSS2]:
       stop_and_go = True
       ret.safetyParam = 53
-      ret.wheelbase = 2.63906
+      ret.wheelbase = 2.67
       ret.steerRatio = 13.9
       tire_stiffness_factor = 0.444  # not optimized yet
       ret.mass = 3060. * CV.LB_TO_KG + STD_CARGO_KG
       if corollaTSS2_use_indi:  # birdman6450#7399's Corolla 2020 TSS2 Tune
+        ret.steerRatio = 15.33
+        tire_stiffness_factor = 0.996  # not optimized yet
```

Comment:
Can you replace the general values for this car if these are more correct? Lines 269 and 270

Reply:
Yes